### ISSUE TITLE
saas-11.5 translations - Error loading Mexican translation

### ISSUE THREAD

**Issue by moylop260** [odoo/odoo#27268]
# Impacted versions:
saas-11.5 (12.0 soon)

# Steps to reproduce:
1. Connect to http://runbot.odoo.com/runbot/1/saas-11.5
2. Use menu -> Setting -> Translations -> Load a translation
3. Choose `Español (México)` language

# Current behavior:
Traceback error
```bash
 File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/addons/account_reports/models/account_financial_report.py", line 1271, in _update_translations
    res = super(IrModuleModule, self)._update_translations(filter_lang=filter_lang)
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/addons/base/models/ir_module.py", line 860, in _update_translations
    self.env['ir.translation'].load_module_terms(mod_names, filter_lang)
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/addons/base/models/ir_translation.py", line 760, in load_module_terms
    tools.trans_load(self._cr, trans_file, lang, verbose=False, module_name=module_name, context=context)
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/tools/translate.py", line 1015, in trans_load
    result = trans_load_data(cr, fileobj, fileformat, lang, verbose=verbose, module_name=module_name, context=context)
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/tools/translate.py", line 1151, in trans_load_data
    irt_cursor.finish()
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/addons/base/models/ir_translation.py", line 151, in finish
    """ % (self._model_table, self._table))
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/home/odoo/runbot/runbot/runbot/static/build/378655-saas-11-5-258178/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: ON CONFLICT DO UPDATE command cannot affect row a second time
HINT:  Ensure that no rows proposed for insertion within the same command have duplicate constrained values.
```

# Expected behavior:
No error

# Video/Screenshot link (optional):
![odoo 12 translation error](https://user-images.githubusercontent.com/6644187/46102918-c8706400-c1cf-11e8-8a9b-2e4f2afab191.gif)

**Comment by odony** [comment]
The crash happens when loading `model` translations in overwrite mode. Can repro in other languages too, e.g. fr_BE.

I noted quite a few POs with entries of `model:ir.ui.view.arch_db` instead of `model_terms:ir.ui.view.arch_db`. Probably requires a re-export/cleanup.

PS: note that POs are typically skipped in forward-ports

**Comment by odony** [comment]
More specifically, [here is a list of modules](https://gist.github.com/odony/50a7782c3dd4fd5e047ef74a93cf8c44) that have a dead POT file lying around (from old rename) and where the POT file still contains `model:ir.ui.view.arch_db` or `model:ir.actions.act_window,help`.

**Comment by moylop260** [comment]
What is the process to fix them?
I create a new PR or I need wait a auto-process from your robots... or
something else?
-- 
Moisés López Calderón
Mobile: (+521) 477-752-22-30
Twitter: @moylop260
hangout: moylop260@vauxoo.com
http://www.vauxoo.com - Odoo Gold Partner
Twitter: @vauxoo

**Comment by mart-e** [comment]
I have fixed it at d9c0dbcd192e1967d9ea81fd984f5489345c1543 in community
Will check the enterprise version too now
